src/main/java/com/avioconsulting/log4j/sqs/processor/FifoMessageProcessor.java

```diff
@@ -21,55 +21,54 @@
  */
 public class FifoMessageProcessor implements LogEventProcessor {
 
-	private static final Logger logger = LogManager.getLogger(FifoMessageProcessor.class);
+    private static final Logger logger = LogManager.getLogger(FifoMessageProcessor.class);
 
-	@Override public MessageRequestWrapper process(ProcessorAttributes processorAttributes) {
-		List<SendMessageRequest> sendMessageRequestList = new ArrayList<>();
-		logger.debug("Fifo - Splitting large message");
-		// Generate Message Hash to use as the group ID
-		UUID uuid = UUID.randomUUID();
-		logger.debug("Large Message UUID: {}", uuid);
-		String[] splitMessage = splitStringByByteLength(processorAttributes.getMessage(), "UTF-8", processorAttributes.getMaxMessageSize());
-		MessageRequestWrapper messageRequestWrapper = new MessageRequestWrapper();
+    @Override
+    public MessageRequestWrapper process(ProcessorAttributes processorAttributes) {
+        List<SendMessageRequest> sendMessageRequestList = new ArrayList<>();
+        logger.debug("Splitting large message");
+        UUID uuid = UUID.randomUUID();
+        logger.debug("Large Message UUID: {}", uuid);
```

Comment:
Can we just combine these into a single debug log that provides the UUID of the large message?

Reply:
Done